Makefile

```diff
@@ -312,7 +313,7 @@ endif
 
 build-releaser: ## - Build a Docker image to run make package including all build tools
 ifeq ($(shell uname -p),arm)
-	$(eval SUFFIX := arm)
+	$(eval SUFFIX := ${CROSSBUILD_ARM_SUFFIX})
```

Comment:
`arm` suffix points to `debian9`, that's how it works in the `g`olang-crossbuild`, see https://github.com/elastic/golang-crossbuild/issues/446 that explains that currently the default debian image is 9.